tests/e2e-cluster.sh

```
@@ -0,0 +1,326 @@
+
+    if [ ! -f "$case_dir/case.env" ] || [ ! -f "$case_dir/buildconfig.yaml" ]; then
+        fail "$name: missing case.env or buildconfig.yaml"
+        return
+    fi
+
+    # Per-case config. case.env references OCP_REGISTRY (exported above).
+    # Reset expectation defaults so cases can omit them.
+    local NAMESPACE BUILD_NAME BUILDER_IMAGE REGISTRY BUILD_STRATEGY OPTIONAL_FLAGS
+    local EXPECT_REGISTERED=false RUN_BUILDRUN=false EXPECT_BUILDRUN=Succeeded BUILD_TIMEOUT=900s
+    # shellcheck disable=SC1090
+    source "$case_dir/case.env"
```

Shellcheck (0.11.0)

[info] Not following: ./case.env was not specified as input (see shellcheck -x).

(SC1091)